spec/views/users/index.html.erb_spec.rb

```diff
@@ -20,13 +21,17 @@
       end
     end
 
-    it 'has an + button to send contact request' do
+    it 'has a + button to send contact request' do
       expect(rendered).to have_button('+', count: users.length)
     end
 
     it 'does have a + button for not requested user' do
       expect(rendered).to have_css("form.button_to[action=\"/users/#{users.second.id}/contact_requests\"]")
     end
+
+    it 'is not possible to see myself in the list of all users' do
```

Comment:
Because this is a view test and you assign only users to the instance variable, it is not possible that the current user is displayed on the view because it isn't in the instance variable.
Because a change in the controller should be tested, it would maybe be better to write a feature spec for this.

Reply:
do you think there should be more covered than is at the moment?

Reply:
I think everything important is covered but maybe with the wrong test type. I would expect a feature test instead of the helper method `users_to_send_requests_to`. But fine for now, can also be refactored later.